app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -323,7 +323,7 @@ class StateFragmentTest {
   }
 
   @Test
-  fun testStateFragment_loadDragDropExp_mergeFirstTwoItems_unlinkFirstItem_worksCorrectly() {
+  fun testStateFragment_loadDragDropExp_mergeFirstTwoItems_invalidAnswer_correctItemCount() {
```

Comment:
fix ktlint issue in this file as well also the imports should be in order with  "java", "javax", "kotlin" and aliases in the end

Reply:
done

Reply:
done